src/codegen/templates/typescript.mustache

```diff
@@ -0,0 +1,20 @@
+import { z } from "zod";
+import { Prefab } from "@prefab-cloud/prefab-cloud-node";
+//import { MustacheStringType } from "./prefab-shared.js";
```

Comment:
I'm curious if this is a left-over or important

Reply:
left-over. I'll remove in following PR